### PR TITLE
[explore] moving the timer from a button to a label

### PR DESCRIPTION
**Pull request by mistercrunch**
screen shot 2016-08-24 at 2 54 19 pm

### CODE REVIEW

File: caravel/assets/stylesheets/less/cosmo/variables.less [hunk 1]
```diff
@@ -14,10 +14,10 @@
 @gray-light:             lighten(@gray-base, 70%);   //
 @gray-lighter:           lighten(@gray-base, 90%); // #eee
 
-@brand-primary:         #515152;
+@brand-primary:         darken(#45A5FF, 25%);
```
Comment (ascott) [inline caravel/assets/stylesheets/less/cosmo/variables.less]: reviewed with this with eli too, let's use the brighter airpal blue. #45A5FF

**Comment by ascott** [comment]
can you make your timer change in another pr? it's a lot easier to track than adding on to my pr.

File: caravel/assets/stylesheets/less/cosmo/variables.less [hunk 2]
```diff
@@ -14,10 +14,10 @@
...
 @brand-success:         #4AC15F;
-@brand-info:            #2AB7CA;
-@brand-warning:         #FED766;
+@brand-info:            lighten(#2AB7CA, 15%);
```
Comment (ascott) [inline caravel/assets/stylesheets/less/cosmo/variables.less]: why are we making these changes?

**Comment by mistercrunch** [comment]
@ascott done!

**Comment by ascott** [comment]
thanks! 🚢

File: caravel/assets/javascripts/explore/components/QueryAndSaveBtns.jsx [hunk 3]
```diff
@@ -14,15 +14,15 @@ export default function QueryAndSaveBtns({ canAdd, onQuery }) {
...
+        <i className="fa fa-bolt"></i> Query
       </button>
       <button
         type="button"
         className={saveClasses}
         data-target="#save_modal"
         data-toggle="modal"
       >
-        <i className="fa fa-plus-circle"></i>Save as
+        <i className="fa fa-plus-circle"></i> Save as
```
Comment (bkyryliuk) [inline caravel/assets/javascripts/explore/components/QueryAndSaveBtns.jsx]: just a question, why do you need a space in the button name?

**Pull request merged**